xfields/ibs/_equilibrium.py

```diff
@@ -253,191 +253,207 @@ def compute_emittance_evolution(
         if iterations > 0:
-            tolerance = np.max(np.abs((current_emittances - previous_emittances) / previous_emittances))
-
-        # Store current emittances for the next iteration
+            tolerance = np.max(
+                np.abs((current_emittances - previous_emittances) / previous_emittances)
+            )
         previous_emittances = current_emittances.copy()
-
-        # Update time step for the next iteration
+        # --------------------------------------------------------------------------
+        # Update time step for the next iteration and increase counter
         time_step = 0.01 / np.max((ibs_growth_rates, twiss.damping_constants_s))
-
         iterations += 1
     # ----------------------------------------------------------------------------------------------
     # We have exited the loop, we have converged. Construct a Table with the results and return it
-    print("\nConverged!")
-    # result_table = Table(
-    #     data={
-    #         "time": np.cumsum(time),
-    #         "gemitt_x": emittances_x_list,
-    #         "gemitt_y": emittances_y_list,
-    #         "gemitt_zeta": emittances_z_list,
-    #         "Tx": Tx,
-    #         "Ty": Ty,
-    #         "Tz": Tz,
-    #     },
-    #     index="time",
-    # )
+    _print(f"Reached equilibrium with tolerance={tolerance:.2e} (vs rtol={rtol:.2e})")
+    result_table = Table(
+        data={
+            "time": np.cumsum(time_deltas),
+            "gemitt_x": np.array(res_gemitt_x),
+            "gemitt_y": np.array(res_gemitt_y),
+            "gemitt_zeta": np.array(res_gemitt_zeta),
+            "Tx": np.array(T_x),
+            "Ty": np.array(T_y),
+            "Tz": np.array(T_z),
+        },
+        index="time",
+    )
     # Provide global quantities as well
-    # result_table._data.update(
-    #     {
-    #         "damping_constants_s": twiss.damping_constants_s,
-    #         "partition_numbers": twiss.partition_numbers,
-    #         "eq_gemitt_x": emittances_x_list[-1],
-    #         "eq_gemitt_x": emittances_y_list[-1],
-    #         "eq_gemitt_x": emittances_z_list[-1],
-    #         "sr_eq_gemitt_x": twiss.eq_gemitt_x,
-    #         "sr_eq_gemitt_y": twiss.eq_gemitt_y,
-    #         "eq_nemitt_zeta": twiss.eq_nemitt_zeta,
-    #     }
-    # )
-    # return result_table
-    return (
-        np.cumsum(time_deltas),
-        res_gemitt_x,
-        res_gemitt_y,
-        res_gemitt_zeta,
-        T_x,
-        T_y,
-        T_z,
+    result_table._data.update(
+        {
+            "damping_constants_s": twiss.damping_constants_s,
+            "partition_numbers": twiss.partition_numbers,
+            "eq_gemitt_x": twiss.eq_gemitt_x,
+            "eq_gemitt_y": twiss.eq_gemitt_y,
+            "eq_gemitt_zeta": twiss.eq_gemitt_zeta,
+            "sr_ibs_eq_gemitt_x": res_gemitt_x[-1],
+            "sr_ibs_eq_gemitt_y": res_gemitt_y[-1],
+            "sr_ibs_eq_gemitt_zeta": res_gemitt_zeta[-1],
```

Comment:
In the end I left `eq_gemitt_[x,y,zeta]` as they are in the TwissTable or it would be too confusing. I have used an ugly but explicit name for our end result.